purchase_invoice_plan/models/purchase.py

```diff
@@ -29,6 +29,20 @@ class PurchaseOrder(models.Model):
         compute="_compute_ip_invoice_plan",
         help="At least one invoice plan line pending to create invoice",
     )
+    ip_total_percent = fields.Float(
+        compute="_compute_ip_total",
+        string="Percent",
+    )
+    ip_total_amount = fields.Monetary(
+        compute="_compute_ip_total",
+        string="Total",
```

Comment:
I think this is giving a warning in the logs and that is why runbot is red:

2021-07-17 08:00:38,675 167 INFO openerp_test odoo.modules.loading: Loading module purchase_order_line_sequence (92/93)
2021-07-17 08:00:39,132 167 INFO openerp_test odoo.modules.registry: module purchase_order_line_sequence: creating or updating database tables
2021-07-17 08:00:39,147 167 WARNING openerp_test odoo.addons.base.models.ir_model: Two fields (ip_total_amount, amount_total) of purchase.order() have the same label: Total.